client/my-sites/subscribers/components/subscriber-list-container/subscriber-list-container.tsx

```diff
@@ -0,0 +1,50 @@
+import { translate } from 'i18n-calypso';
+import Pagination from 'calypso/components/pagination';
+import { EmptyListView } from 'calypso/my-sites/subscribers/components/empty-list-view';
+import { SubscriberList } from 'calypso/my-sites/subscribers/components/subscriber-list';
+import { SubscriberListActionsBar } from 'calypso/my-sites/subscribers/components/subscriber-list-actions-bar';
+import { useSubscriberListManager } from 'calypso/my-sites/subscribers/components/subscriber-list-manager/subscriber-list-manager-context';
+import { Subscriber } from 'calypso/my-sites/subscribers/types';
+import './styles.scss';
+
+type SubscriberListContainerProps = {
+	onClickView: ( subscriber: Subscriber ) => void;
+	onClickUnsubscribe: ( subscriber: Subscriber ) => void;
+};
+
+const SubscriberListContainer = ( {
+	onClickView,
+	onClickUnsubscribe,
+}: SubscriberListContainerProps ) => {
+	const { grandTotal, total, perPage, page, pageClickCallback } = useSubscriberListManager();
+
+	return (
+		<section className="subscribers__section">
```

Comment:
Minor one: as per the style [guideliness](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md) the outermost class name should be: "subscriber-list-container", then you can build up on it like "subscriber-list-container__header", and so on.